fast-list.js

```diff
@@ -72,42 +63,108 @@ function FastList(source) {
   on(this.els.container, 'click', this);
   on(window, 'resize', this);
 
-  this.rendered = schedule.mutation(function() {
-    this.updateContainerGeometry();
-    this.updateListHeight();
+  // Create a list element if one wasn't provided
+  if (!this.els.list) this.els.list = document.createElement('ul');
+  if (!this.els.itemContainer) this.els.itemContainer = this.els.list;
+
+  this.rendered = schedule
+    .mutation(this.renderPhase1.bind(this))
+    .then(this.renderPhase2.bind(this))
+    .then(function() {
+
```

Comment:
But looks like we're still making the list scrollable in "phase 1" but not attaching the the "scroll support" until after "phase 2" (which has a 360ms timeout).
I don't think it can cause big trouble but I'd still feel better enabling scrolling only once we're ready to handle it.